aiogram/api/types/user.py

```diff
@@ -33,6 +41,80 @@ class User(TelegramObject):
 
     @property
     def full_name(self) -> str:
+        """
+        Get full name of user.
+        :return: str
+        """
         if self.last_name:
             return f"{self.first_name} {self.last_name}"
         return self.first_name
+
+    @property
+    def mention(self) -> str:
+        """
+        Get user's username to mention him.
+        Full name will be returned if user has no username.
+        :return: str
```

Comment:
1. Return type is already annotated
1. Does not follow project code-style. When you use annotations in docstring you will need to separate description with annotation with one blank line.

Reply:
> 2. Does not follow project code-style.

It makes sense to configure the linter so that it warns about it. Before each commit, I check the results of tests and syntax analisys of the code.